client/my-sites/hosting/main.js

```diff
@@ -78,7 +79,15 @@ const Hosting = ( { translate } ) => {
 								'Manage your databases with PHPMyAdmin and run a wide range of operations with MySQL.'
 							) }
 						</p>
-						<Button>{ translate( 'Access PHPMyAdmin' ) }</Button>
+						{ ( ! atomicPmaNonce || ! siteId ) && <Spinner /> }
```

Comment:
Something cleaner might be an explicit variable like `isLoadingNonce`, where in connect we might call:

```jsx
const mapStateToProps = ( state ) => {
   const siteId = getSelectedSiteId( state );
   const nonce = siteId && requestAtomicHostingPmaNonce( siteId ).data;
   return {
site: getSelectedSite( state ),
	siteId,
	siteSlug: getSelectedSiteSlug( state ),
	nonce,
    isLoadingNonce: ! nonce || ! siteId;
   };
}
```